main.py

```diff
@@ -46,7 +46,8 @@
 class NoClientNetworkBlocksFound(Exception):
   def __init__(self, provider_name):
     Exception.__init__(self,
-                       'Could not find IP blocks associated with client provider {client_provider}.'.format(
+                       ('Could not find IP blocks associated with client ' +
+                       'provider {client_provider}.').format(
```

Comment:
On the GVC I mentioned I would defer styleguide updates until the next pull, but I want to vet that my understanding on how this should properly be indented is correct, so I've modified it.